p2p/src/network/pubsub/p2p_network_pubsub_actions.rs

```diff
@@ -190,6 +195,17 @@ impl redux::EnablingCondition<P2pState> for P2pNetworkPubsubAction {
                 .topics
                 .get(topic_id)
                 .is_some_and(|topics| topics.contains_key(peer_id)),
+            P2pNetworkPubsubAction::WebRtcRebroadcast { message } => {
+                let source = super::webrtc_source_sk(message)
+                    .public_key()
+                    .peer_id()
+                    .try_into()
+                    .unwrap();
+                pubsub
+                    .mcache
+                    .get_message(&P2pNetworkPubsubMessageCacheId { source, seqno: 0 })
```

Comment:
oh, didn't consider it that we didn't even need to care about seqno since each peer id will be unique, good catch